Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -513,28 +514,51 @@ public static Task TestRegistrationAsync(this UITestContext context, UserRegistr
             "Test media operations",
             async () =>
             {
-                await context.GoToRelativeUrlAsync("Admin/Media");
+                await context.GoToAdminRelativeUrlAsync("/Media");
 
                 context.UploadSamplePngByIdOfAnyVisibility("fileupload");
-                context.UploadSamplePdfByIdOfAnyVisibility("fileupload");
 
-                // Workaround for pending uploads.
+                // Workaround for pending uploads, until you make an action the page is stuck on "Uploads Pending".
                 context.WaitForPageLoad();
                 await context.ClickReliablyOnAsync(By.CssSelector("body"));
 
                 context
-                    .Get(By.CssSelector("#mediaContainerMain tbody tr:nth-child(4) .break-word"))
+                    .Get(By.CssSelector("#mediaContainerMain tbody tr:nth-child(3) .break-word"))
```

Comment:
All these `nth-child` selectors are fragile. E.g. if the OC recipe will include another file, these will need to be reindexed, despite our files still being listed. Use selectors instead that match the content, i.e. the file name, that won't change.